src/python_routing_v02/compute_nhd_routing_SingleSeg_v02.py

```diff
@@ -743,7 +748,7 @@ def compute_nhd_routing_v02(
                         qts_subdivisions,
                         reaches_list_with_type,
                         independent_networks[tw],
-                        param_df_sub.index.values,
+                        param_df_sub.index.values.astype("int64"),
```

Comment:
Sometimes this ends up with floating point values in the index.

Reply:
@jhrehanoaa Will you add a todo for yourself to track down where we might be getting an integer-->float conversion on these index values? It would be (probably) somewhere in our data read functions that we've setup with xarray.